docs/deploy/installation.md

```diff
@@ -1,10 +1,10 @@
 # Load Balancer Controller Installation
 
-!!!warning "Existing AWS ALB Ingress Controller users"
```

Comment:
This is mkdocs annotation for the live docs. Any reason for the change?

The live docs is at https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/deploy/installation/

Reply:
Sorry, I wasn't aware of that! I thought it was strange and it didn't stand out much when looking at it here on Github which is why I changed it.